docs/user_guide/activation_keys/index.md

```diff
@@ -37,11 +37,29 @@ To create a new key,
 - navigate to: Content > Activation Keys
 - click **New Activation Key**
 
-Note the following option:
+![Creating a Host Collection](./activation_key_create.png)
 
-- *Content Host Limit*: This option will control how many Content Hosts may be registered using the key.
+- *Name*: This required option has few restrictions on format. However, since this name is used to identify the activation key to command line tools such as *subscription-manager* including spaces and non-ascii characters will need special consideration in such uses.
 
-![Creating a Host Collection](./activation_key_create.png)
+- *Content Host Limit*: This option will control how many Content Hosts may be registered using the key. An "unlimited" value will not place any limits on usage. Specifying a quantity will limit the number of registered content hosts. Registering with an activation key consumes one of the available limit quantity, while unregistering makes it available again. (ie. This quantity is not a usage counter but a limit of actively registered content hosts.)
+
+- *Description*: A free form description.
+
+- *Environment* and *Content View*: Although optional, at least one activation key used during registration must specify a content view. Taken in order specified to *subscription-manager*, the last activation key with a content view takes precedence.
```

Comment:
The last sentence reads strangely due to starting with 'taken'. Maybe `Subscription manager allows specifying and using more than one activation key at registration. Activation keys are used in the order they are specified to *subscription-manager* and thus the last activation key with a content view takes precedence`.